defaults/main.yml

```diff
@@ -2,3 +2,23 @@
 
 # Specify what volume to put the docker vg on EL systems
 docker_storage_device: ~
+
+# docker daemon remote access
+docker_daemon_remote_access_tls: false
+
+# openssl certs path
+cert_path: /root/ssl_docker
```

Comment:
We possibly should use the default paths of the distribution here to avoid accidental deletion.

Reply:
Both Distribution types have the same path for ssl certs (/etc/ssl). Nevertheless i moved it to os specific vars.